plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -192,11 +193,19 @@ const generateNextReviewAssignments = async ({
 
   console.log('ReviewAssignmentAssignerJoinIds', reviewAssignmentAssignerJoinIds)
 
+  // Remove timeStageCreated from log - to help with tests
+  const reviewAssignmentsLog: ReviewAssignment[] = []
+
+  Object.values(reviewAssignments).forEach((reviewAssignment) => {
+    delete reviewAssignment.timeStageCreated
```

Comment:
hmm, maybe we can do this in tests ?

before expect.toEqual(), can clean up reviewAssignments (with generic method), i.e.:

btw we have lodash now in repo, might be good to use it (https://lodash.com/docs/4.17.15#mapValues, https://lodash.com/docs/4.17.15#omit)

```
removeTimeStageCreated(reviewAssignments: any) => mapValues(reviewAssignments, value => omit(value, ['timeStageCreated']))
...
clearResult(result: any) => ({...result, output: {...output, reviewAssignments: removeTimeStageCreated(result.output.reviewAssignments) }})
...
expect(cleanResult(result)).toEqual({
```

Reply:
Nice. Thanks for the advise. Should be a better use than changing the result on the action.

Reply:
Done: 9e9c78c